internal/providers/terraform/azure/testdata/iothub_test/iothub_test.tf

```diff
@@ -0,0 +1,41 @@
+provider "azurerm" {
+  skip_provider_registration = true
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+# Add example resources for Iothub below
+resource "azurerm_iothub" "example" {
+  name                = "Example-IoTHub"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+
+  sku {
+    name     = "S1"
```

Comment:
We probably should also handle the case with `F1` free tier when the resource has no price and is skipped. Here's an example from another resource:
```go
	if r.SKU == skuFree {
		return &schema.Resource{
			Name:        r.Address,
			IsSkipped:   true,
			NoPrice:     true,
			UsageSchema: LogAnalyticsWorkspaceUsageSchema,
		}
	}
```
https://github.com/infracost/infracost/blob/master/internal/resources/azure/log_analytics_workspace.go#L131-L138